.github/workflows/nightly-checks.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     name: Build packages and upload artifacts
     uses: ./.github/workflows/base-build.yml
 
-  integration-nextjs:
+  integration-nextjs-canary:
```

Comment:
@dimkl I think you also need to update the `needs: integration-nextjs` below

Reply:
yes. this is a change used to check if the `Integration (NextJS canary) / run` would stop being required. I will revert it.